src/image/Placeholder.tsx

```diff
@@ -15,23 +15,33 @@ export const Placeholder = ({
   const blurredSrc = `${imageService}/32x0/filters:blur(10)${path}`;
 
   return (
-    <img
-      {...props}
-      alt=""
+    <div
```

Comment:
I wonder if this additional DOM node could be avoided using `onTransitionEnd` to just set `hidden` attribute on image itself once faded out (or maybe remove whole node)?

```tsx
const [hidden, setHidden] = useState(false)
const handleTransitionEnd = () => { setHidden(true) }
return (<img src={src} hidden={hidden} />)
```

Reply:
It would be nice to not have the additional div, but I couldn't make it work. With the following:
```tsx
<img
  {...props}
  style={{
    position: 'absolute',
    top: '0px',
    left: '0px',
    width: '100%',
    height: '100%',
    objectFit: 'cover',
    objectPosition: 'center center',
    transition: 'opacity 500ms linear',
    opacity: shouldShow ? 1 : 0,
  }}
  alt=""
  aria-hidden
  role="presentation"
  hidden={!shouldShow} // Hide on load
  decoding="async"
  src={blurredSrc}
/>
```
 It flashes the placeholder image on load in reader mode

Reply:
Gave it a spin in #25, what do you think?